Add `Jogja Fun Cup` wiki article (#11660)
ppy/osu-wiki: wiki/Tournaments/JFC/1/en.md

| Referee | ::{ flag=ID }:: [ZStreak](https://osu.ppy.sh/users/2754434), ::{ flag=ID }:: [dhimas arya](https://osu.ppy.sh/users/7108145), ::{ flag=ID }:: [BeefyNoragami](https://osu.ppy.sh/users/11065629), ::{ flag=NL }:: [Miya the turtle](https://osu.ppy.sh/users/13032176) |
| Graphic designer | ::{ flag=ID }:: [hikimaruy](https://osu.ppy.sh/users/9510067) |
| Spreadsheet manager | ::{ flag=HK }:: [ShadeCegLgMn](https://osu.ppy.sh/users/12609866) |
| Wiki editor | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) |

## Links

- **[Master spreadsheet](https://docs.google.com/spreadsheets/d/1hdu0eQKPnA-9y48VV-PWSMDpSDm-Y1e69PoguFraYeE/edit?gid=287204203#gid=287204203)**
- [Forum thread](https://osu.ppy.sh/community/forums/topics/1936900?n=1)
- [Discord server](https://discord.com/invite/kQuJAEbnXe)
- [Livestream channel](https://www.twitch.tv/osuindonesia)

## Participants

> *This list is tentative as of 2024-06-27.*

Reviewer: we don't usually use yyyy-mm-dd in sentences

```suggestion
*This list is tentative as of June 27, 2024.*
```

   - In the Grand Final, the winner of the the upper bracket will only need to win a single match in order to claim the championship title. The winner of the lower bracket, however, will need to win two matches and enforce a *bracket reset* in order to clinch the championship title.
4. Teams who can compete in the next round are determined by:
   - In the Round of 16, each team needs to win 5 points in order to win a match. (Best of 9)
   - In the Quarterfinals and the Semifinals, each team needs to win 6 points in order to win a match. (Best of 11)
   - In both of the Finals weeks, each team needs to win 7 points in order to win a match. (Best of 13)
   - Whether there are teams who are declared to win the match by default.
   - Whether there are teams who are disqualified from the tournament.

### Match regulations

1. Prior to starting the match, representatives from each team must run the `!roll` command in order to determine the banning and picking order.
   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
   - The loser of the `!roll` gets the opposite by default.
   - This rule does not apply in the Qualifier lobbies.

> 2. Each team has to ban **one beatmap** (on the Round of 16) or  **two beatmaps** (from the Quarterfinals onwards) from the corresponding mappool. These beatmaps will not be allowed to be picked by any team during the entire match.

Reviewer: ```suggestion
2. Each team has to ban **one beatmap** (on the Round of 16) or **two beatmaps** (from the Quarterfinals onwards) from the corresponding mappool. These beatmaps will not be allowed to be picked by any team during the entire match.
```

   - Each team **may not ban two beatmaps from the same mod pool**.
   - The tiebreaker map is not allowed to be banned.

> 3. Barring the NoMod pool, each team **may not pick beatmaps from the same mod pool consecutively**.

Reviewer: ```suggestion
3. Barring the No Mod pool, each team **may not pick beatmaps from the same mod pool consecutively**.
```

4. Each team will be given a chance to pick **one warm-up beatmap** (up to 4 minutes max) to be played in the lobby before the match.
   - Playing a warm-up beatmap is not mandatory, and teams may elect to skip their warm-up pick should they wish to.

> 5. In a FreeMod pick, each player has to apply at least one mod to play the beatmap with. Allowed mods are Easy, HardRock, Hidden, or any possible combinations of the three mods.

Reviewer: ```suggestion
5. In a Free Mod pick, each player has to apply at least one mod to play the beatmap with. Allowed mods are Easy, Hard Rock, Hidden, or any possible combinations of the three mods.
```

>    - Each player will have to pick a mod or mod combination that is different to their own teammate's pick. **Playing a FreeMod pick without any mods applied is not allowed.**

Reviewer: ```suggestion
   - Each player will have to pick a mod or mod combination that is different to their own teammate's pick. **Playing a Free Mod pick without any mods applied is not allowed.**
```

   - Players opting to play with Easy will have their scores multiplied by `x1.7` up to a maximum of 850,000 points.

> 6. In the case of a tiebreaker, the tiebreaker map will be played with the FreeMod option enabled. Players are free to play the tiebreaker map with Hidden, HardRock, or Hidden+HardRock should they wish to.

Reviewer: ```suggestion
6. In the case of a tiebreaker, the tiebreaker map will be played with the Free Mod option enabled. Players are free to play the tiebreaker map with Hidden, Hard Rock, or Hidden + Hard Rock should they wish to.
```
